qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/settings/settings_presenter.py

```diff
@@ -129,9 +129,12 @@ def _validate_settings(self):
         self.check_and_populate_with_default("default_peak")
         if not self.settings["default_peak"] in ALL_PEAKS:
             self.settings["default_peak"] = DEFAULT_SETTINGS["default_peak"]
+        self.check_and_populate_with_default("full_calibration")
+        if not path.isfile(self.settings["full_calibration"]):
+            self.settings["full_calibration"] = DEFAULT_SETTINGS["full_calibration"]
```

Comment:
Nice addition - I guess you found it was failing silently\unpleasantly if given an invalid path?